onnx/cpp2py_export.cc

```diff
@@ -81,22 +78,59 @@ PYBIND11_PLUGIN(onnx_cpp2py_export) {
       return *schema;
     });
 
-  m.def("has_schema", [](const std::string& op_type) -> bool {
-      return OpSchemaRegistry::Schema(op_type) != nullptr;
-    });
-
-  m.def("get_all_schemas", []() ->
+  defs.def("get_all_schemas", []() ->
         const std::unordered_map<std::string, OpSchema> {
           return OpSchemaRegistry::registered_schemas();
         });
 
-  m.def("is_attribute_legal", [](const py::bytes& serialized_attr_proto) -> bool {
-       std::unique_ptr<AttributeProto> attr_proto(new AttributeProto());
-       attr_proto->ParseFromString(serialized_attr_proto);
-       return OpSchema::IsAttributeLegal(*attr_proto);
-     });
-
-  return m.ptr();
+  // Submodule `checker`
+  auto checker = onnx_cpp2py_export.def_submodule("checker");
+  checker.doc() = "Checker submodule";
+
+  py::register_exception<checker::ValidationError>(checker,
```

Comment:
when you do this - does the python side inherit from Exception class?

Reply:
yep